docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -215,11 +215,24 @@ Using hyperkit requires having both `hyperkit` and `docker-machine-driver-hyperk
 
 ==== Installing hyperkit
 
-* If you have link:https://docs.docker.com/docker-for-mac/[Docker Desktop for macOS] installed, `hyperkit` is already installed. 
-* If you use link:https://brew.sh[Homebrew] you can install the latest version of `hyperkit`:
+* The following steps explain the installation of the *hyperkit* binary to the *_/usr/local/bin/_* directory:
 
+. Download the *hyperkit* binary using:
```

Comment:
This is a tiny change, but I'd remove "using" here.

```adoc
Download the *hyperkit* binary:
```

Reply:
Done